packages/mui-joy/src/styles/CssVarsProvider.test.tsx

```diff
@@ -18,6 +20,14 @@ describe('[Joy] CssVarsProvider', () => {
       },
       configurable: true,
     });
+    window.matchMedia = () =>
+      ({
+        addListener: () => {},
+        removeListener: () => {},
+      } as unknown as MediaQueryList);
+  });
+  afterEach(() => {
+    window.matchMedia = originalMatchmedia;
```

Comment:
need to reset the `matchMedia`, otherwise, it will affect some tests in the project.